<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Embedded IDE</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="DocumentArea" name="centralWidget"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>&amp;Loggin</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="loggerWidgetContainer">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="tabPosition">
        <enum>QTabWidget::South</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="documentMode">
        <bool>true</bool>
       </property>
       <property name="tabBarAutoHide">
        <bool>true</bool>
       </property>
       <widget class="LoggerWidget" name="loggerCompiler">
        <attribute name="title">
         <string>Compiler Output</string>
        </attribute>
       </widget>
       <widget class="LoggerWidget" name="loggerDebugger">
        <attribute name="title">
         <string>GDB Output</string>
        </attribute>
       </widget>
       <widget class="LoggerWidget" name="loggerApplication">
        <attribute name="title">
         <string>Application Output</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="projectDock">
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Pro&amp;ject</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="ProjectView" name="projectView"/>
  </widget>
  <widget class="QDockWidget" name="debugDocker">
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="DebugUI" name="debugUI"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DocumentArea</class>
   <extends>QWidget</extends>
   <header>documentarea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ProjectView</class>
   <extends>QWidget</extends>
   <header>projectview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LoggerWidget</class>
   <extends>QWidget</extends>
   <header>loggerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DebugUI</class>
   <extends>QWidget</extends>
   <header location="global">debugui.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
